<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSumi</class>
 <widget class="QDialog" name="AddSumi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AÃ±adir Suministro</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Berlin Sans FB Demi</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Seleccione tipo de suministro&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="vacuna">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Vacuna</string>
           </property>
           <property name="icon">
            <iconset resource="Icons.qrc">
             <normaloff>:/6673743_health_hospital_injection_icon.png</normaloff>:/6673743_health_hospital_injection_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="mascarilla">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Mascarilla</string>
           </property>
           <property name="icon">
            <iconset resource="Icons.qrc">
             <normaloff>:/5803988_mask_covid_icon.png</normaloff>:/5803988_mask_covid_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="medicamento">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Medicamento</string>
           </property>
           <property name="icon">
            <iconset resource="Icons.qrc">
             <normaloff>:/2119342_medicines_scientific_icon.png</normaloff>:/2119342_medicines_scientific_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="prueba">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Prueba</string>
           </property>
           <property name="icon">
            <iconset resource="Icons.qrc">
             <normaloff>:/5991186_blood_coronavirus_icon.png</normaloff>:/5991186_blood_coronavirus_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_tipo">
             <property name="text">
              <string>Nombre</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_fecha">
             <property name="text">
              <string>Fecha de Llegada</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_stock">
             <property name="text">
              <string>Stock</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_pais">
             <property name="text">
              <string>Pais/Procedencia</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_dosis">
             <property name="text">
              <string>Dosis/P</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_calidad">
             <property name="text">
              <string>Calidad</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="tipo"/>
           </item>
           <item>
            <widget class="QLineEdit" name="fecha"/>
           </item>
           <item>
            <widget class="QLineEdit" name="stock"/>
           </item>
           <item>
            <widget class="QLineEdit" name="pais">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dosis">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="calidad">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="enviarpushbuton">
         <property name="font">
          <font>
           <family>Berlin Sans FB Demi</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
